### ISSUE TITLE
Version number

### ISSUE THREAD

**Issue by rmcgibbo** [choderalab/pymbar#162]
```
rmcgibbo@DN0a22f7a9  ~/projects/pymbar (3.4.2) ‹master›
$ python setup.py install
[... snip ...]
1.3.1-py3.4.egg/setuptools/dist.py:283: UserWarning: The version specified requires normalization, consider using '2.1.0b0' instead of '2.1.0-beta'.
[... snip ...]
```

PEP440 version numbers: https://www.python.org/dev/peps/pep-0440/#pre-releases

**Comment by kyleabeauchamp** [comment]
We're close to 3.0 release anyway, so we might just fix it then...

**Comment by kyleabeauchamp** [comment]
Fixed by #163
